hawkbit-core/src/main/java/org/eclipse/hawkbit/repository/DistributionSetFields.java

```diff
@@ -25,6 +25,14 @@
      * The description field.
      */
     DESCRIPTION("description"),
+    /**
+     * The createdAt field.
+     */
+    CREATEDAT("createdAt"),
```

Comment:
I'd prefer "CREATED_AT" and "LAST_MODIFIED_AT"

Reply:
While I agree, that it would improve readability, it would be inconsistent with the other field enums (cf. TargetFields).

Reply:
Ok, makes sense... thanks!